### ISSUE TITLE
[CAIA Intake] Transition Experience: V1

### ISSUE THREAD

**Issue by nkontrabecki** [department-of-veterans-affairs/va.gov-team#83802]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: Transition Experience
- **OCTO product owner:** Lisa Koenigsberg
- **Product name:** Transition Experience
- **Product manager:**  Gene Dreyband
- **Slack channel:** [#ves-transition](https://dsva.slack.com/archives/C06C7RS27GD)
- **Dedicated content writer on your team (if you have one):** Ian Seabock/iseabock-va and Aaron Kriegler/vetskrieg
- **Dedicated a11y specialist on your team (if you have one):** `Insert name and/or GH handle`
 
Transitioning Servicemembers (TSMs) and Recently Separated Veterans (RSVs) are overwhelmed by the transition process. The transition process itself is a major risk for creating vulnerable populations. In addition to juggling many personal life challenges as they move into civilian life, TSMs and RSVs have no easy way to find the range of benefits and which they are eligible for.

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [x] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

The primary goal of the Veteran Transition Support program is to serve as a hub that can help the discovery of benefits, and ideally simplify the enrollment process for benefits with similar eligibility requirements. The goal is increasing the utilization of benefits, and increasing the proportion of Veterans that engage with VA post-service.

While the core value proposition is for TSMs and RSVs, the Veteran Transition Support project will also be useful for any Veteran re-engaging with VA, regardless of their time since service.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [x] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Initial Discovery - Complete
Design Intent touchpoint - scheduled for 5/24/2024
User flow and Wireframes prepared for Design Intent touchpoint

We plan to develop and conduct limited user testing staging late June to early July 2024.

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [x]  Yes
- [ ]  No
 
 *Note: **If you check yes, we’ll reach out to discuss details about the content in the react widget. We use these widgets to display entry points for new products to a certain percentage of users who visit our static pages. Please refer to this [GitHub reference](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md) for dynamic content.
 
- [x]  I acknowledge that if I change the launch to be incremental, or change it from incremental to 100%, I must notify CAIA of this change as there is additional work involved that may impact deadlines.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** [82863](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82863)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/vet-transition-support) 
- [User Flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1716219020147/013c6f25f45e35963469c7bb7a94b66e1402f663?sender=uba527cafb8ded20512886129)
- [Wireframes](https://www.figma.com/design/Y72reZ0Qt9xteezdnLbBDW/Wireframes-for-v1?node-id=0%3A1&t=l90aXJCrJfAktf1O-1)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

**Comment by nkontrabecki** [comment]
> Noting here that we'll need to hold on CAIA involvement until OCTO has weighed in

Is there still a hold on CAIA involvement until OCTO has weighed in? Just wanted to understand if there is anything additional that our team needs to provide to the CAIA team to include any meetings we need to meet the CAIA touchpoint requirements in the Collab Cycle.

**Comment by nkontrabecki** [comment]
@kristinoletmuskat Is there any specific guidance for URL naming convention for our product on VA.gov?

**Comment by nkontrabecki** [comment]
Good Morning. Our team has some questions. We are available for a call to discuss the details of our questions below:

1. Where should our product live on the [VA.gov](http://va.gov/) website?
          This will inform both our URL path and the breadcrumbs that appear on each page.
2.  What would CAIA recommend re: how users find our product?
3. What's the process for confirming the wording across all screens of our v1 product?
4. We want to use action-oriented language for the heading of our questionnaire pages. Currently, the heading says "Complete this questionnaire to find relevant benefits and resources"
          What does the CAIA team think about that wording?
5. Does CAIA have any recommendations for indicating that our product will be updated in the future?
           One possible strategy is using "still in beta testing" language on our welcome page, like the [VA chatbot page](https://www.va.gov/contact-us/virtual-agent/)]

**Comment by kristinoletmuskat** [comment]
Hey @nkontrabecki, apologies for the delay! Confirmed with Mikki we can release our hold on this.

I can pick up this IA work next week and get back to you on your IA questions for sure. Our content team is inundated with work at the moment (and always) so for questions #3-5, @strelichl do you have any sense of timing?

**Comment by kristinoletmuskat** [comment]
Hi @nkontrabecki , we have some IA thinking for your team to review in [your user flow mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1716219020147/013c6f25f45e35963469c7bb7a94b66e1402f663?wid=17-1719339080528) (I wanted to put this info somewhere where y'all could easily find it, but happy to move it to a different place that works better). 

 I am meeting with our IA OCTO lead Mikki this afternoon to go over this, and welcome your team's feedback (as comments or stickies) in the meantime. And if you are uncertain what all this info means or how to use it, I'm happy to meet to walk through it.

**Comment by kristinoletmuskat** [comment]
Hi @nkontrabecki and @cora-goldston-bah -- just a note that our OCTO leads are going to do some thinking on how to message this work to stakeholders at OPIA and VEO. So, we'd like to explore some home page placement options, but can't confirm yet whether that is a go! Let me know if you have any Qs. That decision shouldn't impact your URLs or breadcrumbs.

**Comment by cora-goldston-bah** [comment]
@kristinoletmuskat - quick question:

Do you know if the following disclaimer will be required on our questionnaire? It's included in the Forms Library template, but since users aren't applying for benefits through our product, we want to confirm if this is needed or not:

"Note: According to federal law, there are criminal penalties, including a fine and/or imprisonment for up to 5 years, for withholding information or for providing incorrect information (See 18 U.S.C. 1001)."

**Comment by nkontrabecki** [comment]
Good Afternoon @kristinoletmuskat, Our team reviewed your IA ideas/considerations in our [v1 user flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1716219020147/013c6f25f45e35963469c7bb7a94b66e1402f663?sender=ue28fdc8b087c2343f2b72871). We understand that you are recommending that we use [va.gov/benefit-eligibility-questionnaire/](https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fva.gov%2Fbenefit-eligibility-questionnaire%2F&data=05%7C02%7C%7C8a1cf6e517334910ea8a08dc96b60242%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C638550955669479275%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5fowBp8hWZtxGvWdBKBDNCGmjYBfI5VEPS9Am8bfpXU%3D&reserved=0) as the URL for our tool. We will discuss with our PM if this URL path is acceptable and we will contact you with any concerns.

@YevgenyDreyband @cora-goldston-bah @iseabock @iseabock-va @vetskrieg @lkoenigsberg

**Comment by kristinoletmuskat** [comment]
@cora-goldston-bah we do not need that disclaimer on this questionnaire, since the user isn't applying for anything from the VA.

What would be great to include is a disclaimer about how this is a self-service tool that does not in any way guarantee any benefits or start a claim -- I don't think we have a template for that type of disclaimer, but the content team should be able to help you out once they have capacity to pick up this work!

**Comment by nkontrabecki** [comment]
@kristinoletmuskat We are interested in hearing what your team discovers as a result of exploring some home page placement options and entry points for our product. Please let me know when you and your team are available to meet with us to discuss the entry point options. Thank you in advance.

**Comment by kristinoletmuskat** [comment]
Hey! Just FYI that I will be OOO Friday July 12- Monday July 22. If anything comes up during that that's urgent, feel free tor each out to @jennymayoco

**Comment by cora-goldston-bah** [comment]
> Hey! Just FYI that I will be OOO Friday July 12- Monday July 22. If anything comes up during that that's urgent, feel free tor each out to @jennymayoco

Thank you for the heads-up!

**Comment by nkontrabecki** [comment]
@kristinoletmuskat See below for a list of items for discussion at our tag up this afternoon. We look forward to speaking with you. Please feel free to forward the invite to anyone who may be able to contribute to the conversation! Thank you

1. Points of Entry
- We are interested in hearing what your team discovers as a result of exploring some home page placement options and entry points for our product.
- The CAIA team recommended several points of entry to our tool. Below are Figma mockup links for the following points of entry:

- [ ] [VA.gov](http://va.gov/)[ homepage](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2546-7365&t=L9plQbUYKZquQMDE-4)
- [ ] [v0 Resources & Support page](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2546-7380&t=L9plQbUYKZquQMDE-4)
- [ ] [The existing servicemember hub](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2546-7458&t=L9plQbUYKZquQMDE-4)
- Who will be responsible for linking to our tool from the points of entry?
- Additionally, CAIA mentioned linking from the Welcome Kit landing page and/or PDF to our tool. However, from CAIA's notes on the user flows, it sounds like there are differing opinions about linking from the Welcome Kit.
- We need to confirm the appropriate points of entry and determine how to implement these links.

2. Confirm the wording for the v1 screens
3. We want to use action-oriented language for the heading of our questionnaire pages. Currently, the heading says "Complete this questionnaire to find relevant benefits and resources"
What does the CAIA team think about that wording?Does CAIA have any recommendations for indicating that our product will be updated in the future?
4. One possible strategy is using "still in beta testing" language on our welcome page, like the [VA chatbot page](https://www.va.gov/contact-us/virtual-agent/)

**Comment by kristinoletmuskat** [comment]
Notes from our sync:

- We only need 1 entry point to launch. Mikki (CAIA OCTO) is meeting with OPIA and VES on 7/22 to discuss home page and other placement options that require outside-VFS approval
- How do we get SME approval? Team is not sure. 
- Will include cross-agency benefits in the future in this tool
- Content timing: they're putting together their research plan right now, but don't have dates yet. Research is probably about 2-3 weeks out (last week of July, first week of August)
- the MVP will only show some benefits at first, so we want to make it clear to users that it will improve and change over time

**Next steps:**
- Laura will get back to team with an estimate of when we might be able to get to the content work
- team will look into who their SMEs are (for approval)

**Comment by laurwill** [comment]
Hi @nkontrabecki @cora-goldston-bah , I'm still going through the Figma and leaving content comments, but wanted to note a couple high-level things here:
- For the name of the tool, this seems to be what's usually called either a "benefits explorer" or "benefit finder" tool (like [this example from benefits.gov](https://www.benefits.gov/benefit-finder)). I think we might want to use one of those here, and then make the H1 of the tool screens "Find benefits" or "Explore benefits" rather than using a noun phrase with "recommendations." We'll also want to adjust some of the language on the intro and results screens so that it's clear we're not exactly recommending things — more like "find benefits you may be eligible for." Let's chat more about this!
- Now that we're seeing the scope of the MVP version (education and career benefits only), I think we'll need to revisit the entry points plan for MPV launch — if the tool will only talk about 1 or 2 benefit types, we'll want the entry points to be from pages or page sections specific to those benefit types, so that we're setting the right expectations for what the tool will do. Then when you launch a more complete version, we can give it more prominent placement on homepage or other more general pages. So for example, rather than placing the "Try me" info alert at the top of the Getting started with VA page, we'd probably place it in the "Go back to school or start a new career" section.
- If the tool is only for service members and Veterans, we'll need to make this clearer in the entry points or tool content or both, so that family members and caregivers know it won't work for them.
- If you have the logic for how all the answer options affect the results, that would be really helpful to see — for example, I wasn't sure how the different "goals" selections would change what we're showing people for benefit results. 

I think we're finding time to sync this week, but let me know if you have questions in the meantime, and looking forward to collaborating more on this!

**Comment by laurwill** [comment]
Oh and just noting that I made a few comments in the [desktop version](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=0-1&t=qMmI18e7UZJA18WX-0) but most in the [mobile views](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2963-22830&t=qMmI18e7UZJA18WX-0) — it's easier to assess header and paragraph length in mobile, so thanks for adding those!

**Comment by kristinoletmuskat** [comment]
Hey everyone, I heard from Laura that we're planning to launch this tool with just the career/employment benefits to start. Given that change, we will likely do the following related to IA:

1. **Continue with the same plan for the URL**: ie put the URL at the root level (meaning va.gov/[tool-name-tbd]) rather than the careers level (va.gov/careers/[tool-name-tbd]). We want to do this so that as we add benefit areas to the tool in the future, we wont have to continuously move the URL around (which would require a bunch of redirect work). 
2. **Change the entry point plan**: Before, we were considering home page and other high-level placements for links to this tool. With the careers focus, we're now planning to link to this tool just from the careers benefit hub and left nav.

To finalize the URL and link labels, we'll need to coordinate with @laurwill on the H1 for this first iteration AND the intended H1 for the tool once it includes all benefits. 

I've updated the [mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1716219020147/013c6f25f45e35963469c7bb7a94b66e1402f663?wid=0-1722446695647) and published an [IA spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/v1-transition-experience-questionnaire.md) with this info. 

Does this make sense to everyone? Any Qs? Happy to discuss in a quick call to if that's easier!

**Comment by laurwill** [comment]
Hi @cora-goldston-bah @RonnieYoung-UX @YevgenyDreyband ,
Also wanted to follow up on some of the content conversations we were having in the Figma file. Did you all want to chat more about any of the suggestions I made in there?

We'd also love to talk more about the content model for this product, to understand better how you're tying different goals, etc, to the results that appear at the end — is the Excel sheet Gene shared the source of truth for this kind of thinking, or do you have somewhere else we should look?

And we understand you want to build out the filter and sort functionality for future iterations, but since the MVP will have only one type of benefit, we wondered if it would make more sense to wait on the filter and sort, to avoid showing people features they don't really need in that version. 

If you are moving forward with the sort/filter, Christina Gednalske on Modernized Check-In team recently did a study on a combined sort/filter feature, so she'd be a good resource if you haven't already connected with her. 

Anyway, let us know if you want to chat about any of this or if you have updated designs to share. Thanks!

**Comment by cora-goldston-bah** [comment]
Hi @kristinoletmuskat and @laurwill, apologies for the delay!

We'd love to reconnect about the content and IA for our v1 product. I think Nicole is coordinating with you to schedule a time. Depending on the timing, @RonnieYoung-UX might be on the call without me, but she is fully immersed in the work and will be ready to talk about it.

Laura, can you remind us which Excel spreadsheet was shared with you? I think we have a newer version, but want to confirm.

Nicole will also share our updated Figma mockups to inform the conversation. The updated mockups will include the full list of benefits that will be shown for v1.

Thank you and our team's looking forward to reconnecting with you!

**Comment by laurwill** [comment]
Hey @cora-goldston-bah , 

This is the spreadsheet we had started looking at: [Benefit and Resource Mapping for v1.xlsx](https://dvagov.sharepoint.com/:x:/r/sites/VESPortfolioOpsManagement/Shared%20Documents/Product%20Lines/Digital%20Experience%20Product%20Line/Transition/Other/Benefit%20and%20Resource%20Mapping%20for%20v1.xlsx?d=w2265597fce2a494480cbbc71d6db0840&csf=1&web=1&e=iP158u)

Let us know if there's a more current version! And if you have a link to the updated Figma, I'd love to take a peek before we meet tomorrow if I have time.

**Comment by laurwill** [comment]
Hi @cora-goldston-bah @RonnieYoung-UX @nkontrabecki , documenting some updates on this work. Apologies for the delay in getting these in the ticket! cc @kristinoletmuskat 

- We're now meeting on a biweekly basis to discuss updates to the V1/V2 tool. We've gone through several rounds of collaboration/iteration on the tool UI and the schema.
- We've discussed options for incremental release of the tool, and you'll confirm with us when you decide if that's what you want to do.
- You took a version of the tool to midpoint review last Friday 11/15 that didn't have all the most recent content suggestions included — let's check in during our sync today about midpoint feedback and how to incorporate that with my [most recent round of comments in the Figma file](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT/v2-Prototype-Design?node-id=6634-11526&node-type=canvas&t=zhkxTKWGs1m3i1f7-0), so we can get you ready for staging.
- I still need to pass you a final set of updates to benefit descriptions but want to confirm — does the results page in the Figma file I linked here show all the current versions of all the cards in the tool?

**Comment by cora-goldston-bah** [comment]
Hi @laurwill , thank you for documenting the updates here!

For the third bullet about incorporating content suggestions:
To keep our current production schedule, we have to freeze our code by December 2nd, which means any "Must" content changes need to be incorporated in our Figma file and then incorporated on staging before that.

I know you're planning to apply the Must/Should/Consider framework to your feedback (thank you!). If we can get your Must/Should/Consider prioritization by **the end of this week (November 22)**, that would be ideal! Monday will probably work too, but it gets dicey after that because of Thanksgiving.

Of course, we will still discuss and consider any feedback that's not a "Must" before launch. We're on a tighter timeline for our first launch to production, but we can certainly make updates as a "fast follow" to the live version.

And to answer your question - yes, the Results page in the Figma file reflects the current versions of our benefit cards. One very minor exception - we're fixing a capitalization issue on the VALife card this afternoon. But otherwise, the cards are up-to-date.

If any of the updated descriptions will be "Must" changes, please let us know. Thanks again for your great content recommendations!

**Comment by laurwill** [comment]
Also just found this "no results" error in staging with a typo and some other issues that we need to update. Not sure where this is in Figma, so just noting here!
![Image](https://github.com/user-attachments/assets/14acf3bc-c96e-4ca8-95bd-ae39f3b9331a)

Suggested changes:
**No results**
We didn't find any results that match your answers and any filters you added. If you added filters, try removing them and updating your results list. Or you can review or change your answers.
Review or change your answers [link]

We're also planning to add more benefits and resources to this tool. Check back soon to find more benefits you may want to apply for.

**Comment by nkontrabecki** [comment]
Good Afternoon @laurwill I will address 2 comments:

- **Have SMEs approved the tool logic? (meaning how you're assigning results to various combinations of answers)** Gene and Nicole are working with Megan Kelley and her team to set up discussions with the SMEs to confirm the logic. Gene and Nicole have also been in contact with Danielle Thierry to communicate the status.

- **For non-VA benefits that don't have existing approved VA.gov content, have SMEs from the partner organizations approved the content on the result cards?** Gene and Nicole will seek formal review and approval from Bill and Jon on the 4 non-VA benefits included in our next version: (Skillbridge, TAP, VetSuccess on Campus, Veterans' Preference in federal hiring).

**Comment by nkontrabecki** [comment]
- Is Figma still the content source of truth, as opposed to [staging](https://staging.va.gov/benefit-eligibility-questionnaire/introduction)? There are lots of small content differences, and want to make sure I'm reviewing the exact content you're planning to use for launch.

Yes, Figma is the content source of truth and please note that the Staging site is still under development.

- Does staging have the full logic implemented for all the results and scenarios? I'm seeing results that don't seem to make sense based on my answers, so if it does have everything implemented, we should talk about what I'm seeing. And if it doesn't, I'd love to retest when it's updated just to make sure I understand how it will work at launch.

@cora-goldston-bah  and/or @RonnieYoung-UX  will respond to this comment as soon as they investigate.

**Comment by cora-goldston-bah** [comment]
@laurwill  Thank you for sharing your questions!

Regarding your second question about whether the logic on staging:

The logic is fully implemented on staging. For the Burial example that you shared in our Slack thread, **it is working as intended**, but we're planning 2 future updates that will populate more benefits in that situation.

Currently, specific Goals are mapped to specific benefits, as shown in the Q1 column of [the schema](https://dsva.slack.com/archives/C07LSPBFY1Z/p1729623575373729) that we previously shared. Per our discussions, we pared down the list of Goals, and "Plan for my and my family's future" replaces 2 old goals (Carry out legacy planning and Commemorate my service) that were more focused on burials and memorials. Currently, burial is currently the only benefit associated with the goal "Plan for my and my family's future."

We'll likely need to prioritize errors and bug fixes before Monday's code freeze, our Staging Review, and initial launch. However, we do want to address this in 2 ways for our next version:

- Changing our goals-to-benefit mapping so that choosing your goals doesn't reduce the number of benefits shown at the end. Rather, we want to adjust so your chosen goals will push the aligned benefits to the top, but you'll still be able to see other benefits that align with your other answers. 
    - We're thinking something like a "Recommended just for you" section for benefits that align to your chosen goals and a "You also may qualify for this" section for benefits that align to your other answers, but not the goals you selected.
- Holistically reviewing and re-mapping our goals list to the benefits. We recognize that "Plan for my and my family's future" is a much broader goal than the burial-focused goals it replaced. Therefore, we agree it makes sense to map this goal to more benefits. For our next version, we'll want to re-review and assess which benefits should be associated with which goals.

If you tested other staging scenarios that you're concerned about, please send them our way! They might be addressed by the longer-term fixes mentioned above, but we also want to ensure the logic is working as currently designed.

Also, we are reviewing your identified "musts" in the Figma file — we'll close the loop tomorrow about those.

**Comment by laurwill** [comment]
Also noticed that the filter list in staging has some capitalization issues and includes a second housing type that shouldn't be there, I think: 
![Image](https://github.com/user-attachments/assets/42f1f155-b8d8-4e72-b40d-736ed29b107c)

These should all be in sentence case, and I think the only housing type should be "housing assistance," so we'd need to remove "loan guaranty."

**Comment by cora-goldston-bah** [comment]
Hi @laurwill, we wanted to provide an update about reviewing and incorporating your feedback!

We are working to incorporate the "must" items that you identified in Figma, the items you've mentioned above, and a short-term fix for the "Plan for my and my family's future" goal mapping issue that you identified. (As mentioned in my previous message, we're also planning 2 future changes that will address this issue in the long term.)

Today, we learned that the VA Platform team is implementing a holiday code freeze for this Thursday and Friday. We thought we'd have until Monday to update the staging site before our Staging Review code freeze, but apparently that isn't the case. Tomorrow, our developers need to focus their efforts on 2 launch-blocker fixes identified in our Midpoint Review.

What that means for us — we will continue working on your identified "must" changes, but they won't be visible on the staging site until after our Staging Review on Friday, December 6th.

We're still working to incorporate your "must" items before we launch to production. We're working on them now, but we also have some time after Staging Review to finish incorporating changes.

We do have questions about a few of the "must" items:

* [Comments on the VA mental health benefit card:](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT?node-id=6634-11886#1036454959)
    * We absolutely agree about removing the word "click" — that change has already been made on staging, and we will update Figma this week to match.
    * However, the rest of the description matches the [v2 schema](https://dsva.slack.com/archives/C07LSPBFY1Z/p1729623575373729) that we shared back in October, and we didn't receive any feedback about changing the description then.
    * We're hesitant to wordsmith the description right now, particularly because we agree with [your comment](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT?node-id=6634-11886#1036467819) about applying consistency in whether we're including CTAs in the description or not.
        * Making the language consistent across all cards will likely take a significant amount of time and collaboration, so we think it's more appropriate for a "v2.1" follow-up.
        * Does that plan work for you?
* [Comments on the Apply for a discharge upgrade benefit card:](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT?node-id=6634-11886#1036352368)
    * Again, we absolutely agree about removing the word "click" — that change has already been made on staging, and we will update Figma this week to match.
    * However, we think this card needs some additional information beyond the revised wording about answering questions and getting step-by-step instructions.
        * When we’ve conducted user testing, Veterans and transitioning service members seemed pleasantly surprised that you can apply for a discharge upgrade.
        * Many of our users may not know what this is, so we want to keep some text explaining what happens if you’re awarded an upgrade.
        * Are you opposed to us keeping something about potentially being eligible for more benefits if you're awarded a discharge upgrade? Maybe instead, we can tweak the wording of this second sentence?
* [Comment about the Sort dropdown on the Results and window-shopping pages:](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT?node-id=6634-11718#1036335724)
    * Adopting the recommended change will cause a different problem:
        * Since both sort options would start with “Alphabetically by benefit,” people are likely to make an error and select the wrong one.
    * I think we should give ourselves more time to discuss and adjust the wording together.
        * This has implications not only for the Sort dropdown options, but also for the dynamic sentence that says “Showing ## results, filtered by x, sorted by...”
        * We'd like to continue discussing options with you and implement new wording for a "v2.1" follow-up.
        * Does that plan work for you?


Please let us know your thoughts. We know you're traveling this week, so we can work async through GitHub and Slack this week. We also have a biweekly tag-up on Tuesday, so we can discuss sticky questions then as well.

Safe travels! Wishing you a great holiday weekend :)

**Comment by laurwill** [comment]
Thanks @cora-goldston-bah ! Responses here:
- [Comments on the VA mental health benefit card:](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT?node-id=6634-11886#1036454959)
    - I missed this one in the schema in my first review, my apologies — I've realized that the spreadsheet format is not the best way for me to review content like this, so I'll make sure to find a better format for my reviews going forward.
    - The comments about the description aren't at the level of wordsmithing — I think this is a must change because, as I understand it, you're limiting content on these cards about VA benefits to approved live content from VA.gov. It looks like you instead pulled the content from the [legacy mentalhealth.va.gov page](https://www.mentalhealth.va.gov/). Is the link on the card also going to the legacy page? Since this is a VA.gov tool, we need to link to modernized VA.gov pages for any benefit type that has a modernized page. And we need to match the content on the modernized page we link to for accuracy and consistency. 

- [Comments on the Apply for a discharge upgrade benefit card](https://www.figma.com/design/4J2vqMA2VY6wqgcjvhF2aT?node-id=6634-11886#1036352368)
     - Sure, we can include additional content. But the sentence on the card ("If your application goes through and your discharge is upgraded, you'll be eligible for the VA benefits you earned during your period of service.") has language we'd need to change for site style — passive voice, an idiom "goes through", and "you'll be eligible" instead of our standard "you may be eligible" to avoid confirming eligibility that may not be accurate. So we'd need to say something more like "If we approve your discharge upgrade, you may be eligible for additional VA benefits." 
     - I think any net-new card content that goes beyond approved live VA.gov content needs SME approval, right?

- Sort dropdown:
     - I'm okay leaving this for V2. With so few benefits in the list , I don't think sort/filter is vital to the UX of this version. Did participants interact with the filter or sort during usability testing? 
     - We could do "Alphabetically by name" and "Alphabetically by type" to avoid the repeat "benefit," or potentially "A to Z by name" and "A to Z by type" — we should check how we phrase this in other sort options on VA.gov, I can't remember which phrasing we've used.

**Comment by cora-goldston-bah** [comment]
One quick follow-up:

Our VA mental health benefit card does link to the [updated mental health page](https://www.va.gov/health-care/health-needs-conditions/mental-health/) on VA.gov. But we see your comment about updating the description and we're adding that task to our queue!

**Comment by cora-goldston-bah** [comment]
Hi @laurwill, hope you had a great holiday!

Don't be alarmed if you get several Figma notifications today. I'm just documenting in Figma what we've discussed for addressing your identified "must" feedback. Apologies in advance for any notification emails!

**Comment by nkontrabecki** [comment]
> Also just found this "no results" error in staging with a typo and some other issues that we need to update. Not sure where this is in Figma, so just noting here! ![Image](https://github.com/user-attachments/assets/14acf3bc-c96e-4ca8-95bd-ae39f3b9331a)
> 
> Suggested changes: **No results** We didn't find any results that match your answers. If you added filters, try removing the filters. Or you can review or change your answers. Review or change your answers [link]
> 
> We're also planning to add more benefits and resources to this tool. Check back soon to find more benefits you may want to apply for.

[Jira ticket #332 ](https://jira.devops.va.gov/browse/PTEMSVT-332) Was created to address this feedback.

**Comment by nkontrabecki** [comment]
> * **For the "save results" part of the experience,** I noticed in staging that the "Go back to review your entries" link goes nowhere. It looks like the link doesn't save the person's answers, it just brings them to a results list with specific results. Is that right? We'll need to make clear what the "save" link actually saves, and we'll need to remove the "Go back" link from saved results pages if it doesn't work.

Hi @laurwill  [Jira ticket #285](https://jira.devops.va.gov/browse/PTEMSVT-285) was created to address the "save results", "Go back to review your entries" link that goes to nowhere. Fixed in Sprint 16.

**Comment by jessicawoodin** [comment]
Hi @mistymg! I'm the new designer on the Transition Experience team. I think we can go ahead and close out this ticket :)

**Comment by mistymg** [comment]
::waves enthusiastically:: Hi old friend!! @jessicawoodin
